src/icalendar/cal.py

```diff
@@ -17,9 +17,13 @@
 from icalendar.timezone_cache import _timezone_cache
 
 import pytz
+import six
 import dateutil.rrule
 from pytz.tzinfo import DstTzInfo
 
+if six.PY3:
+    unicode = str
```

Comment:
and that should be as simple as doing ``from icalendar.compat import unicode_type`` and than use unicode_type instead of unicode.